src/number_theory/divisors.lean

```diff
@@ -398,4 +400,77 @@ begin
   exact nat.lt_succ_of_le (nat.divisor_le (nat.mem_divisors.2 ⟨ha, h⟩))
 end
 
+
+
+/-- # Prime divisors -/
+
+lemma dvd_of_mem_factors {n p : ℕ} (h : p ∈ n.factors) : p ∣ n
+  :=
+begin
+  by_cases hn : n = 0,
+  { convert dvd_zero p },
+  { rwa ←mem_factors_iff_dvd (pos_iff_ne_zero.mpr hn) (prime_of_mem_factors h) }
+end
+
+lemma prime_divisors_eq_to_filter_divisors_prime (n : ℕ) :
+n.factors.to_finset = (divisors n).filter prime
```

Comment:
I think all the lemmas except this one probably belong in `data/nat/prime`, since they're all about `nat.factors` not `divisors`.

Reply:
Yeah, I wondered whether they belonged somewhere else now but I wasn't sure where.  I'll move them over.